common-arcana.lic

```diff
@@ -72,35 +72,42 @@ module DRCA
     end
   end
 
-  def start_barb_abilities(skills, settings)
-    skills
+  def start_barb_abilities(abilities, settings)
+    # Note, you must know Power meditation or Powermonger mastery
+    # for your active abilities to be detected by DRSpells.
+    abilities
       .reject { |name| DRSpells.active_spells[name] }
-      .each do |name|
-        activate_barb_buff?(name)
-        waitrt?
-        pause settings.meditation_pause_timer.to_i if get_data('spells').barb_abilities[name]['type'].eql?('meditation')
-        DRC.fix_standing
```

Comment:
Moved to end of the method rather than after each activation of an ability

Reply:
Is this desirable? What if you have meditation, something else, meditation. Would you want to stand up in between the two meditations?

Reply:
Resolved per https://github.com/rpherbig/dr-scripts/pull/4772/commits/714857a3b33c36cb86d9209d30d39a7a3f831dd3